app/views/errors/auth_error.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2 class="heading">Session Expired</h2>
+<p>Your current session has expired and we can't authenticate your request. Try logging in again, refreshing the page, or clearing your cache if you continue to experience problems.</p>
```

Comment:
On the maintenance page, we link to instructions for clearing your browser cache. We should probably do that here, too: `<a href="http://kb.iu.edu/data/ahic.html">clearing your cache</a>`